pkg/controller/compute/managed_test.go

```diff
@@ -254,6 +254,11 @@ func TestCreate(t *testing.T) {
 			},
 			want: want{
 				err: errors.Wrap(errBoom, errCreateAKSCluster),
+				ec: managed.ExternalCreation{
+					ConnectionDetails: map[string][]byte{
+						"password": {},
```

Comment:
How does this end up being empty? Don't we generate the password if it's `""`?

Reply:
The `newPasswordFn` used in this test returns an empty string.